.gitignore

```diff
@@ -6,4 +6,5 @@ test/output/
 examples/example-options.json
 .idea
 /composer.phar
-test.php
\ No newline at end of file
+test.php
+composer.lock
```

Comment:
I removed composer lock because it caused issues with phpunit and different Php versions. See https://travis-ci.org/SparkPost/php-sparkpost/builds/151201635